ext/include/opentelemetry/ext/zpages/latency_boundaries.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+// include libraries
+#include <chrono>
+#include <climits>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace ext
+{
+namespace zpages
+{
+
+/**
+ * Latency_Boundary_Name enum is used to index into the Latency_Boundaries vector that is declared later on,
+ * using this enum lets you access the Latecy_Boundary at each index without using magic numbers
+ */
+typedef enum LatencyBoundaryName
+{
+  k0MicroTo10Micro = 0,
+  k10MicroTo100Micro,
+  k100MicroTo1Milli,
+  k1MilliTo10Milli,
+  k10MilliTo100Milli,
+  k100MilliTo1Second,
+  k1SecondTo10Second,
+  k10SecondTo100Second,
+  k100SecondToMax
+} LatencyBoundaryName;
+
+/**
+ * kLatency_Boundaries constant that contains the 9 latency boundaries and enables them to be iterated over
+ */
+const std::vector<std::chrono::nanoseconds> kLatencyBoundaries = {
+    std::chrono::nanoseconds(0),
```

Comment:
Nit: Add 'using std::chrono::nanosecodns' after the include statements to avoid repeating it here.